test/intl402/DateTimeFormat/prototype/formatRange/date-x-greater-than-y-throws.js

```diff
@@ -2,9 +2,9 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-description:
+description: |
   Throws a RangeError if date x is greater than y.
-info:
+info: >
```

Comment:
Thanks for the quick fix.

Just a nit: use > for description and | for info. The former will ignore line breaker while the latter will preserve them in the parsed values.

Reply:
Whoops, can do! I actually just mimicked the the other files in these two directories (from #2139). I can update all of them accordingly though.